src/shared/components/pipeline-run-logs/StatusIcon.tsx

```diff
@@ -66,17 +66,26 @@ export const StatusIconWithText: React.FC<StatusIconProps & { text?: string }> =
   ...others
 }) => {
   return (
-    <Badge isRead>
+    <Label>
       <span
+        className="pf-u-mr-xs"
         style={{
           color: status
             ? getRunStatusColor(status).pftoken.value
             : getRunStatusColor(runStatus.Cancelled).pftoken.value,
         }}
       >
         <StatusIcon status={status} {...others} />
-      </span>{' '}
-      {text ?? status}
-    </Badge>
+      </span>
+      <span
+        style={{
+          color: status
+            ? getRunStatusColor(status).pftoken.value
+            : getRunStatusColor(runStatus.Cancelled).pftoken.value,
+        }}
+      >
```

Comment:
Refer my previous comment.  Once color prop is added on the label, you no longer need to set the color on icon, text seperately.

Reply:
updated
<img width="1383" alt="Screenshot 2022-10-10 at 6 33 30 PM" src="https://user-images.githubusercontent.com/24852534/194877189-d3ee0fa5-f3be-459a-8df6-081cef864030.png">